src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -342,15 +343,35 @@ public Response setMetadataRoot( @PathParam("identifier")String dvIdtf, String b
     
     @GET
     @Path("{identifier}/facets/")
+    /**
+     * return list of facets for the dataverse with alias `dvIdtf`
+     */
     public Response listFacets( @PathParam("identifier") String dvIdtf ) {
-        return allowCors(response( req -> ok(
-                        execCommand(new ListFacetsCommand(req, findDataverseOrDie(dvIdtf)) )
```

Comment:
Good point - I'll switch it to `ListFacetsCommand` (assuming that's not the source of the problem).